spec/facility_spec.rb

```diff
@@ -24,3 +26,153 @@
     end
   end
 end
+
+RSpec.describe Facility do
+  before(:each) do
+    @facility_1 = Facility.new({name: 'Albany DMV Office', address: '2242 Santiam Hwy SE Albany OR 97321', phone: '541-967-2014' })
+    @facility_2 = Facility.new({name: 'Ashland DMV Office', address: '600 Tolman Creek Rd Ashland OR 97520', phone: '541-776-6092' })
+
+    @cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+    @bolt = Vehicle.new({vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev} )
+    @camaro = Vehicle.new({vin: '1a2b3c4d5e6f', year: 1969, make: 'Chevrolet', model: 'Camaro', engine: :ice} )
+  end
+  
+  describe '#register_vehicle' do
+    it 'can register a vehicle' do
+      @facility_1.add_service('Vehicle Registration')
+      expect(@cruz.registration_date).to eq(nil)
+      expect(@facility_1.registered_vehicles).to eq([])
+      expect(@facility_1.collected_fees).to eq(0)
+
+      @facility_1.register_vehicle(@cruz)
+      expect(@facility_1.registered_vehicles).to eq([@cruz])
+      expect(@cruz.registration_date).to eq(Date.today)
+      expect(@cruz.plate_type).to eq(:regular)
+    end
+  end
+  
+  describe '#collected_fees' do
+    it 'can collect fees' do
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(@cruz)
+      expect(@facility_1.collected_fees).to eq(100)
+    end
+  end
+
+
+  describe 'multiple vehicles test' do
+    it 'can register and collect fees fot multiple vehicles' do
+      
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(@cruz)
+      @facility_1.register_vehicle(@camaro)
+      
+      expect(@facility_1.registered_vehicles).to eq([@cruz, @camaro])
+      
+      @facility_1.register_vehicle(@bolt)
+      expect(@facility_1.collected_fees).to eq(325)
+    end
+  end
+  
+  describe "can't register vehicle" do
+    it 'will not register vehicle if not offered at facility' do
+      expect(@facility_2.registered_vehicles).to eq([])
+      expect(@facility_2.services).to eq([])
+      
+      @facility_2.register_vehicle(@bolt)
+      
+      
+      expect(@facility_2.registered_vehicles).to eq([])
+      expect(@facility_2.collected_fees).to eq(0)
+
+    end
+  end
+end
+
+
+
+RSpec.describe Facility do
```

Comment:
You don't need to do `RSpec.describe ClassName` except for the very first block. I think you might be trying to separate it out for each type of behavior that you're testing, in this case the Written Test. Instead you can do something like this (nested inside the one original `RSpec.describe` block):
```
describe "written test" do
 ...
end
```